applications/CompressiblePotentialFlowApplication/python_scripts/initialize_geometry.py

```diff
@@ -0,0 +1,252 @@
+        default_parameters = KratosMultiphysics.Parameters( """
+            {
+                "model_part_name": "insert_model_part",
+                "skin_model_part_name": "insert_skin_model_part",
+                "geometry_parameter": 0.0,
+                "maximum_iterations": 1,
+                "print_output": false,
+                "remeshing_flag": false,
+                "initial_point": [0,0],
+                "metric_parameters":  {
+                    "minimal_size"                         : 5e-3,
+                    "maximal_size"                         : 1.0,
+                    "sizing_parameters": {
+                        "reference_variable_name"               : "DISTANCE",
+                        "boundary_layer_max_distance"           : 1.0,
+                        "interpolation"                         : "constant"
+                    },
+                    "enforce_current"                      : true,
+                    "anisotropy_remeshing"                 : false,
+                    "anisotropy_parameters": {
+                        "reference_variable_name"          : "DISTANCE",
+                        "hmin_over_hmax_anisotropic_ratio"      : 0.5,
+                        "boundary_layer_max_distance"           : 1,
+                        "interpolation"                         : "Linear"
+                    }
+                },
+                "distance_modification_parameters":{
+                    "distance_threshold"                     : 0.001,
+                    "check_at_each_time_step"                : true,
+                    "avoid_almost_empty_elements"            : true,
+                    "deactivate_full_negative_elements"      : true
+                }
+            }  """ );
+        settings.ValidateAndAssignDefaults(default_parameters)
+        self.model=Model
+        self.main_model_part = Model.GetModelPart(settings["model_part_name"].GetString()).GetRootModelPart()
+        self.geometry_parameter = settings["geometry_parameter"].GetDouble()
+        self.initial_point = settings["initial_point"].GetVector()
+        self.skin_model_part_name=settings["skin_model_part_name"].GetString()
+
+        '''Remeshing loop parameters'''
+        self.do_remeshing = settings["remeshing_flag"].GetBool()
+        self.step = 0
+        self.max_iter = settings["maximum_iterations"].GetInt()
+        self.print_output_flag = settings["print_output"].GetBool()
+
+        self.metric_parameters = settings["metric_parameters"]
+        self.distance_modification_parameters = settings["distance_modification_parameters"]
+
+        KratosMultiphysics.VariableUtils().SetNonHistoricalVariableToZero(MeshingApplication.METRIC_TENSOR_2D, self.main_model_part.Nodes)
+
+        ''' Defining linear solver to be used by the variational distance process'''
+        import python_linear_solver_factory #Linear solver for variational distance process
```

Comment:
```suggestion
        from KratosMultiphysics import python_linear_solver_factory #Linear solver for variational distance process
```
the old way of importing stuff will stop working eventually, please always use the same as when you imports stuff from the CPS app